frame/vesting/src/benchmarking.rs

```diff
@@ -375,6 +375,49 @@ benchmarks! {
 		);
 	}
 
+	force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 0 .. T::MAX_VESTING_SCHEDULES;
+
+		let source: T::AccountId = account("source", 0, SEED);
+		let source_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(source.clone());
+		T::Currency::make_free_balance_be(&source, BalanceOf::<T>::max_value());
+
+		let target: T::AccountId = account("target", 0, SEED);
+		let target_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(target.clone());
+		
+		// Give target existing locks
+		add_locks::<T>(&target, l as u8);
+		
+		let per_block = T::MinVestedTransfer::get();
+		let transfer_amount = per_block.checked_mul(&20u32.into()).unwrap();
+		// 2 x transfer_amount because we will create 3 vesting schedules and remove one
```

Comment:
Why *3* vesting schedules?

Reply:
That is the configured number for MAX_VESTING_SCHEDULES for tests, but I have updated this part to include variable that calculates number of expected vestings after removal

Reply:
@NZT48 you have to write benchmarks for a generic runtime. Yes the test environment may only have 3, but then a different runtime would maybe have more or less. You should always retrieve these values from the generic arguments.

Reply:
Update to determine last vesting schedule index from MAX_VESTING_SCHEDULES that is read from runtime.